### PR TITLE
feat(PairedButtons): Adding paired buttons feature

### PR DESCRIPTION
**Pull request by Coloringfood**
Adding Paired Buttons. Adding a clear all to SelectInput

### CODE REVIEW

File: src/components/SelectInput/SelectInput.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import ReactDOM from 'react-dom';
+import styled from 'styled-components';
```
[the remark this answers is not in the document]
Comment (Coloringfood) [reply src/components/SelectInput/SelectInput.js]: Done

File: src/components/PairedButtons/PairedButtons.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+import Button from '../Button';
+
+const ALIGN_RIGHT = 'align_right';
+const ALIGN_LEFT = 'align_left';
+
+const ButtonsContainer = styled.div`  
+  display: grid;
+  grid-template-columns: ${props => {
+    let result = '';
+    for (let i = props.count; i > 0; i--) {
```
Comment (MorganNesbitt) [inline src/components/PairedButtons/PairedButtons.js:13]: can this happen in react? think its probably better to not do this type of logic in styled components

File: src/components/PairedButtons/PairedButtons.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+function renderButton ({onChange, selected, options, id, disabled, onDarkBg}) {
+  return function renderButtonHandler (option, index) {
+    const align =
+      index === 0
+        ? ALIGN_LEFT
+        : index === options.length - 1
+        ? ALIGN_RIGHT
+        : '';
+    let otherProperties = {};
```
Comment (MorganNesbitt) [inline src/components/PairedButtons/PairedButtons.js:49]: think these should be declarative on the component as well 
<StyledButton
outline={option.value !== selected}
/>

File: src/components/PairedButtons/PairedButtons.js (hunk 2 above)
Comment (Coloringfood) [reply src/components/PairedButtons/PairedButtons.js:13]: Would you suggest I pass in the full text string as a prop then?

Comment (MorganNesbitt) [reply src/components/PairedButtons/PairedButtons.js:13]: i would say something like <ButtonsContainer gridTemplateColumns={'1fr '.repeat(options.length).trim()} />
then styled component would be
grid-template-columns: ${props => props.gridTemplateColumns}

**Pull request merged**